README.md

```diff
@@ -116,9 +116,12 @@ _options:_
 * -i    file path to the XLSX for use with xlsx2json
 * -D    (optional) path to the sheets folder; if no folder is specified the the current working directory is used
 
-_[\<sheet name\> ...]:_
```

Comment:
This is still supported in xlsx2json command. Since two commands differ more significantly now I suggest splitting README into two subheadings, one focused on `json2xlxs` command and other one for `xlsx2json` command.

Reply:
It's not, the code doesn't do anything with it but I will split anyway.

Reply:
It is, it just does not use same file helper.

See `const sheets = args._.length > 0 ? args._ : converter.allSheets();`

Reply:
Ah yeah sorry I am blind